packages/calypso-e2e/src/lib/blocks/block-flows/form-patterns.ts

```diff
@@ -0,0 +1,113 @@
+import {
+	ExpectedFormField,
+	labelFormFieldBlock,
+	makeSelectorFromBlockName,
+	validatePublishedFormFields,
+} from './shared';
+import { BlockFlow, EditorContext, PublishedPostContext } from '.';
+
+interface ConfigurationData {
+	labelPrefix: string;
+	patternName: string;
+	otherExpectedFields: ExpectedFormField[];
```

Comment:
**question:non-blocking** - I went the other way and defined an additional parameter in the constructor `validationData` in some of the block tests I've written.
Example: https://github.com/Automattic/wp-calypso/pull/81065/files#diff-415c2abb9f3f69f190ceba55b72002f402ed52da4c321e0b10bb2e5ce3c2bac4R36


What do you think about clearly separating the `configurationData` and `validationData`?

Reply:
Oh sure, that's a great idea! 😄 I'll pull the `otherExpectedFields` to a new interface 👍